libs/common/src/lib/select-account-kb/select-kb/select-kb.component.scss

```diff
@@ -74,9 +74,19 @@
       color: $color-text-menu-option-disabled;
       pointer-events: none;
     }
+    .delete-btn {
+      visibility: hidden;
```

Comment:
better to use `opacity: 0`, so you can add a `transition: opacity $transition-hint` as well